From 2b38f74b07b55ac989423c9154fa5a3d81b7e9ec Mon Sep 17 00:00:00 2001
From: Carl Worth <cworth@cworth.org>
Date: Wed, 22 Oct 2008 16:17:39 -0700
Subject: [PATCH] Use consistent idiom for obtaining static_state

This doesn't make any difference, but it's cleaner to have
each function follow the same idiom for obtaining these pointers.
---
 src/i965_render.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/i965_render.c b/src/i965_render.c
index 9d2896a..a9d7f66 100644
--- a/src/i965_render.c
+++ b/src/i965_render.c
@@ -1287,8 +1287,8 @@ i965_composite(PixmapPtr pDst, int srcX, int srcY, int maskX, int maskY,
 {
     ScrnInfoPtr pScrn = xf86Screens[pDst->drawable.pScreen->myNum];
     I830Ptr pI830 = I830PTR(pScrn);
-    gen4_static_state_t *static_state = pI830->gen4_render_state->static_state;
     struct gen4_render_state *render_state = pI830->gen4_render_state;
+    gen4_static_state_t *static_state = render_state->static_state;
     Bool has_mask;
     Bool is_affine_src, is_affine_mask, is_affine;
     float src_x[3], src_y[3], src_w[3], mask_x[3], mask_y[3], mask_w[3];
-- 
1.6.0.3

